physionet-django/user/forms.py

```diff
@@ -930,3 +931,29 @@ def save(self):
         TrainingQuestion.objects.bulk_create(training_questions)
 
         return training
+
+
+class OrcidRegistrationForm(RegistrationForm):
+    """
+    Form to register new user after signing in with ORCID.
+    This saves user as the same way RegistrationForm but also stores
+    orcid_token and
```

Comment:
"but also stores orcid_token and..."?

Reply:
I think I got distracted and forgot to end this sentence. Fixed!